src/Fable.Transforms/Python/Python.fs

```diff
@@ -43,6 +43,16 @@ type Expression =
         upper: Expression option *
         step: Expression option
 
+type Literal =
+    | FloatLiteral of float
+    | IntLiteral of obj
```

Comment:
@dbrattli 

Should it not be of type `int`?

Reply:
@MangelMaxime No it was intentionally since we need to store every kind of int from (u)int8 to (u)int64 and even BigInteger. Int will not do since we will get both sign and overflow problems

Reply:
Oh I see